DependencyInjection/Configuration.php

```diff
@@ -40,6 +52,7 @@ public function getConfigTreeBuilder()
                 ->scalarNode('namespace')->defaultValue('Application\Migrations')->cannotBeEmpty()->end()
                 ->scalarNode('table_name')->defaultValue('migration_versions')->cannotBeEmpty()->end()
                 ->scalarNode('name')->defaultValue('Application Migrations')->end()
+                ->booleanNode('profiler_enabled')->defaultValue($this->debug)->end()
```

Comment:
``enable_profiler`` or ``profiling`` would be more consistent with naming used in existing bundles (DoctrineBundle is using ``profiling``)